components/EpisodesList/EpisodeItem.js

```diff
@@ -1,32 +1,56 @@
+import { useAtom } from "jotai";
+import { getFormattedDate, getCoverURL } from "../Episode/EpisodeFunctions";
+import { checkFavorites, toggleFavorites } from "../Favoring/FavoringFunctions";
+import { initialFavorites } from "../Favoring/initialFavorites";
 import SVGIcon from "../Icons";
 import {
-  FavButton,
-  StyledEpisodeLink,
-  StyledListItem,
+  ListButton,
+  EpisodeLink,
+  OverviewListItem,
+  PreviewImage,
+  OverviewText,
 } from "./EpisodesList.styled";
 
-export default function EpisodeListItem({
-  episodeNumber,
-  title,
-  href,
-  parts,
-  onHandleFavorites,
-  isFaved,
-}) {
+export default function EpisodeListItem({ episode }) {
+  const {
+    nummer: number,
+    titel: title,
+    veröffentlichungsdatum: releasedate,
+    teile: parts,
+    links,
+  } = episode;
+
+  const [favorites, setFavorites] = useAtom(initialFavorites);
+  const isFaved = checkFavorites(favorites, number);
+
   return (
-    <StyledListItem>
-      <StyledEpisodeLink href={href}>
-        <p>
-          #{episodeNumber} ...{title}{" "}
-          {parts?.length > 0 ? "(Spezialfolge)" : ""}
-        </p>
-      </StyledEpisodeLink>
-      <FavButton type="button" onClick={onHandleFavorites}>
+    <OverviewListItem>
+      <EpisodeLink href={`/episodes/${number}`}>
+        <PreviewImage
+          src={getCoverURL(links)}
+          alt="bla"
+          width={400}
+          height={400}
+        />
+        <OverviewText>
+          <p>
+            {number} - {title}
+          </p>
+          <p></p>
```

Comment:
Empty element, please delete. Use styling instead if you wanted to get some margin here.